src/app/hooks/useAsiointiKieli.ts

```diff
@@ -1,14 +1,22 @@
 import { useQuery } from '@tanstack/react-query';
 import { client } from '../lib/http-client';
 import { configuration } from '../lib/configuration';
+import { Language } from '../lib/localization/localization-types';
 
-export const getAsiointiKieli = () => {
-  return client.get(configuration.asiointiKieliUrl);
+export const getAsiointiKieli = async (): Promise<Language> => {
+  const response = await client.get(configuration.asiointiKieliUrl);
+  return response.data ?? 'fi';
 };
 
 export const useAsiointiKieli = () =>
   useQuery({
     queryKey: ['getAsiointiKieli'],
     queryFn: getAsiointiKieli,
+
     staleTime: Infinity,
   });
+
+export const useUserLanguage = (): Language => {
+  const { data } = useAsiointiKieli();
+  return data || 'fi';
+};
```

Comment:
Näitä muutoksia ei varmaankaan tarvita. useAsiointiKieli-hookkia ei pitäisi tarvita käyttää muualla kuin Wrapper-komponentissa, koska asiointikieli asetetaan i18nextiin jo ennen kuin mitään muuta renderöidään -> voidaan siis käyttää i18nextin kieltä. Olisi myös ehkä parempi tehdä oma `useTranslation`-hook, joka palauttaa myös `translateName`:n, jolle olisi jo annettu i18nextin kieli. Näin komponenteissa ei tarvitsisi ensin hakea kieltä ja sitten kutsua sillä translateName:a.

Kielen saa i18nextistä näin:
```js
const { i18n } = useTranslation();
const { language } = i18n;
```